services/frontend/src/components/profileForms/qualificationsForm/educationForm/EducationFormView.jsx

```diff
@@ -276,20 +276,36 @@ const EducationFormView = ({
                     )}
                   </Form.Item>
                   {/* Checkbox if event is on-going */}
-                  <Form.Item
-                    fieldKey={[fieldElement.fieldKey, "ongoingDate"]}
-                    initialValue={false}
```

Comment:
You can use this I believe. It will help you avoid rapping the entire thing in the conditional statement and it will prevent repeated code:

 `initialValue={educationItem ? undefined : false}`

Reply:
I tried it and I managed to reproduce the same warning:
![20](https://user-images.githubusercontent.com/46769064/134986575-a08db61a-beeb-4fd7-b83b-98932269cda3.JPG)
However, the error didn't occur right when the page was loaded like before. It came after I tried creating and saving various education entries. I'm going to do some debugging to see if its the same problem as before or a new one.

Reply:
I cant seem to recreate the error with the proposed changes. do you have step-by-step recreation instructions by any chance?

Reply:
After debugging, I discovered this error is sligtly different: when you delete an entry from the Education or Experience form and then add another one right away, the warning appears. It happens for both my code and yours, so I'll have to try and find a workaround to that.